src/components/GeoMap.js

```diff
@@ -473,7 +472,7 @@ export default class GeoMap {
         } else {
           const originalValueRaw = d.rollupValue[attrKey];
           const originalValue = (attrValue.transform) ?
-              attrValue.transform(originalValueRaw) : originalValueRaw;
+              attrValue.transform(originalValueRaw, this.maptable.rawData) : originalValueRaw;
```

Comment:
@melalj  - okied dokie!  docs are updated.  i got a little turned around with all the different groupby, transform and rollup declarations so lemme know if they don't look right to you!